example/vm/config.yaml

```diff
@@ -0,0 +1,201 @@
+# Last Collector-Contrib Validation: v0.96.0
+receivers:
```

Comment:
There's a couple other receivers and comments in our `otel-cloud-stack` chart that seem unrelated to k8s. I'm wondering if they apply to this config

1. support for OTel-Arrow: https://github.com/lightstep/otel-collector-charts/blob/main/charts/otel-cloud-stack/values.yaml#L64-L73
2. filelog receiver for logs https://github.com/lightstep/otel-collector-charts/blob/main/charts/otel-cloud-stack/values.yaml#L578-L648 I mean a bunch of the attributes that are k8s-specific wouldn't apply here, but otherwise seems like it might be important for collecting logs? https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver

Reply:
I think we can hold off on arrow for now. let's do filelog in a followup, there's enough in this PR to be happy with and filelog will need some specific tuning.